### ISSUE TITLE
unsafe variance ignore leaves spurious notes

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#444]
```py
from typing import Generic

from basedtyping import T_co

class A(Generic[T_co]):
    def foo(self, t: T_co):  # type: ignore[unsafe-variance]
        pass
```
```
test.py: note: In member "foo" of class "A":
test.py:6:22: note: If this is intentional and you know what you are doing, you can ignore this line with 'unsafe-variance'
test.py:6:22: note: Error code "misc" not covered by "type: ignore" comment
Success: no issues found in 1 source file   
```

# workaround

Also ignore `misc`

**Comment by KotlinIsland** [comment]
While we're here we can clean up the message a little
